### DISCUSSION TITLE
difficulty level not changeable

### DISCUSSION THREAD

**Discussion by Tigertye** [CorsixTH/CorsixTH#2225] [Research Room]
Hi Guys!

I decided to try out the campaign just for fun today and the Difficulty says Doctor (Medium) which I assume is the default but I can't change it from that. Is it not working or do I just not know how to do it?

Thanks,
TT

**Comment by lewri** [comment]
Difficulty selector is changed by dropdown. Changes in 0.66 now mean it will dismiss if clicked outside of 
Edit; wrong webm, sorry

[screen-recording(4).webm](https://user-images.githubusercontent.com/20030128/187169110-68213e47-c041-4995-8305-778d61d06545.webm)

**Reply by Tigertye** [reply]
lewri yer awesome. thankyou!
is there any way to do it in a single scenario too?

**Reply by Tigertye** [reply]
:( mine does not do that. I have no drop down menu.

**Reply by Tigertye** [reply]
I also notice that mine does not have a savegame version at the bottom. is that also a part of v66? mine only says v66 at the bottom.

how did you take a video? i can't find directions for that.
also is there a way to make the game save the screenshots as a jpeg instead of a bmp?

**Reply by lewri** [reply]
I used a webm recorder addon in Firefox for the video.
What version are you using and what os? Did it come from a repository?

**Reply by lewri** [reply]
The screenshot query already has an issue open here
https://github.com/CorsixTH/CorsixTH/issues/1862

**Reply by Tigertye** [reply]
I am using v66 and I got from here like a week ago tops. I run it in Win7 Pro 64-bit. 
I wasn't able to change that setting in v60 either.

That OS does not have access to the internet because I hate cooties.

**Comment by SilPho** [comment]
This might be because you're using Theme Hospital files from the unpatched version of the original game, maybe from a CD rip instead of a download from GOG or Origin.

Check your Theme Hospital files in the HOSP/LEVELS folder, if you only see FULLXX.SAM and LEVEL.LX files then you have the unpatched version, but if you have EASYXX.SAM and HARDXX.SAM files then you do have the patched version and it should be working in Corsix.

**Reply by lewri** [reply]
Can confirm, if the EASY or HARD files are missing, the dropdown will lose said difficulty.

**Reply by Tigertye** [reply]
nope, no easy or hard files. How is this possible?
I installed the game files from the CD because I played them. I installed the pregnancy patch and that other patch. I have all the map-pack files I could find. 
Since the day I first installed the orig game in '99 I have never been able to select the Difficulty level. But then I never knew what to look for either.

I'll be back...

**Reply by SilPho** [reply]
![image](https://user-images.githubusercontent.com/22282491/187491550-1fc91252-caff-4b66-8681-ec861d93bd65.png)
It was fairly obvious in the original game if it was patched correctly. The up and down arrows would cycle through the options.

Are you using the game files from an original CD? Because I'm not sure how you would run the patch for that on a modern 64-bit system.

One quick way to get your hands on the patched version of the game is to buy it from Good Old Games. I don't know where you are, but if you're in the UK it's on sale right now for only £1.29
https://www.gog.com/game/theme_hospital

**Reply by Tigertye** [reply]
Oh now there's a screen I haven't seen in a very very long time :)

Yes, I am using the files from an original CD. And no, when I extracted the patch files, like the readme said to, I still did not get the dropdown menu. However I am stubborn and, luckily, somewhat fanatical when I backup stuff to switch to a new HD. I found the folder that had the copy of an old Theme Hospital install and I just copied the Levels folder contents over to my current HOSP/LEVELS folder. I now have dropdown abilities! I tried this because my current Levels folder only had 82 files while that old one had 87. I assumed it had whatever the patch file was missing and I went for it. Turns out my assumption was right this time.

I'm in the USA but thanks for offering that option.

So is there a way to choose the Difficulty level when playing a Single Scenario? Because that would be fantastic.

**Reply by SilPho** [reply]
Very glad to hear you've got it working :D 

At the moment Single Scenarios only have one difficulty setting because difficulty is controlled by each level file, not by the game itself, if that makes sense. A scenario author could make multiple versions of their scenarios if they wanted to.

**Reply by Tigertye** [reply]
it is? oh I love tweaking my level files. I'll have to look into that further.

I spent half the night playing in hard mode. To be honest it plays just like medium did lol. I did try out the Roujin's code and I like that one :grin: 

You guys are so great! I'm the only person I know who's even heard of this game let alone plays it.
Have a happy!
TT

**Reply by Tigertye** [reply]
hmmm. Ok, I've given a few level files a good scouring and I'm not seeing it. Would I be correct in assuming you meant the difficulty of single scenarios is more about the parameters put forth in the levels file rather than a simple game setting?

**Reply by SilPho** [reply]
Exactly. This is why there are 3 different files for each level in the original campaign, one for each difficulty option. There is no such thing as a difficulty option, levels are just easier or harder based on how much things cost, how competant the staff are, which diseases are available, and lots of other factors.
